testing/integration/package_version_test.go

```diff
@@ -130,6 +130,13 @@ func TestComponentBuildHashInDiagnostics(t *testing.T) {
 		allHealthy,
 		5*time.Minute, 10*time.Second,
 		"agent never became healthy. Last status: %v", &stateBuff)
+	t.Cleanup(func() {
+		if t.Failed() {
```

Comment:
Isn't this supposed to be `!t.Failed() then t.Log`? Otherwise it will print the log if a test passes.

Reply:
fixed